core/src/stored/device_resource.cc

```diff
@@ -278,6 +278,17 @@ bool DeviceResource::Validate()
     my_config->AddWarning(
         "Setting 'Maximum Block Size' on a non-tape device is unsupported");
   }
+
+  if (max_concurrent_jobs == 0) {
+    my_config->AddWarning(
+        "Max Concurrent Jobs default value is now 0. It will be changed to 1 "
+        "starting from Bareos 23.");
+  } else if (max_concurrent_jobs > 1 && dev_type != DeviceType::B_TAPE_DEV) {
+    my_config->AddWarning(
+        "Setting 'Maximum Concurrent Jobs' on a disk device to a value higher "
+        "than 1 is not recommended. Please set it back to 1.");
```

Comment:
```suggestion
        "Setting 'Maximum Concurrent Jobs' on a non-tape device to a value higher "
        "than 1 is not recommended, because it reduces performance on restore.");
```